web/satellite/src/store/modules/usage.ts

```diff
@@ -111,6 +113,10 @@ export const bucketUsageModule = {
             let result = await fetchBucketUsages(projectID, before, state.cursor);
             if (result.isSuccess) {
                 commit(BUCKET_USAGE_MUTATIONS.FETCH, result.data);
+
+                if (result.data.totalCount > 0) {
```

Comment:
Should we move this logic into a mutation since its changing the state?

Reply:
yes. Good point